### ISSUE TITLE
Does CAT12 pipeline works?

### ISSUE THREAD

**Issue by iPsych** [robdahn/boney#10]
In boney_preprocessing file
      case {2,'cat'} % not working

It seems CAT12 integration is still not fully implemented, right?
Can I use already processed CAT12 result from outside now? or CAT implementation is not yet developed at all?

**Comment by iPsych** [comment]
Below is full error log.
It seems like SPM or CAT12 version incompatibility.
Can you specify exact versions of those software you are using ?
I guess that It would be easier to use that specific version instead of debugging code accord to CAT12/SPM12 update.

`
SPM12: v7771                                       12:58:43 - 26/09/2024
========================================================================
Initialising SPM                        :                   ........done

SPM present working directory:
	/media/user/disk/Boney

SPM12: spm_boney (v0.1)                            12:58:55 - 26/09/2024
========================================================================
    
    Boney Toolbox
    Version 0.1 alpha 



------------------------------------------------------------------------
26-Sep-2024 12:59:27 - Running job #1
------------------------------------------------------------------------
26-Sep-2024 12:59:27 - Running 'Bone segmentation'
  0 of 1 SPM segmentations can be used. 
  1 of 1 files need preprocessing. 
Item opts: No field(s) named
ngaus
warpreg
redspmres
biasstr
accstr
Item registration: No field(s) named
T1
brainmask
cat12atlas
darteltpm
shootingtpm
regstr
bb
vox
Item extopts: No field(s) named
segmentation
surface
admin
Item atlases: No field(s) named
aal3
mori
anatomy3
julichbrain
Tian_Subcortex_S4_7T
Schaefer2018_100Parcels_17Networks_order
Schaefer2018_200Parcels_17Networks_order
Schaefer2018_400Parcels_17Networks_order
Schaefer2018_600Parcels_17Networks_order
Item GM: No field(s) named
warped
Item WM: No field(s) named
warped
Item atlas: No field(s) named
warped
dartel
Item bias: No field(s) named
native
dartel
26-Sep-2024 12:59:28 - Failed  'Bone segmentation'
Error using spm_jobman>fill_run_job (line 472)
No executable modules, but still unresolved dependencies or incomplete module inputs.
In file "/media/ipsych/EVEESS_10TB_15/Boney/spm12/spm_jobman.m" (v7744), function "fill_run_job" at line 472.
In file "/media/ipsych/EVEESS_10TB_15/Boney/spm12/spm_jobman.m" (v7744), function "spm_jobman" at line 247.
In file "/media/ipsych/EVEESS_10TB_15/Boney/spm12/toolbox/boney/boney_segment_preprocessing.m" (???), function "boney_segment_preprocessing" at line 89.
In file "/media/user/disk/Boney/spm12/toolbox/boney/boney_segment.m" (???), function "boney_segment" at line 128.

The following modules did not run:
Failed: Bone segmentation`

**Comment by robdahn** [comment]
No, it was already working (at least under some conditions) but I hadn't removed the comment ;-)

I did not test CAT so often as it uses nearly the same segmentation as SPM for the head classes with some correction those are mainly to optimise the brain brain classes. Relevant differences, as I was able to see so far, are given in case of children when the SPM registration failed. Moreover, I observed various problems with the head/bone segmentation in CAT that is coming from the focus on brain tissue (what I have to fix some day). 

I have just tested some of the settings again to avoid problems with the different expertgui settings in CAT.
